### ISSUE TITLE
Slow installation

### ISSUE THREAD

**Issue by aminya** [jwlawson/actions-setup-cmake#35]
When I use this package it takes around 30s to install CMake. Could you speed up this installation (e.g. by skipping some steps like getting the version info)

**Comment by hartwork** [comment]
For anyone interested in faster approaches: what libexpat is doing to install ancient CMake at https://github.com/libexpat/libexpat/pull/536/files takes 1-2 seconds runtime. That approach is more low-level and less flexible though, to be fair.

**Comment by jwlawson** [comment]
Yeah it would definitely be faster to guess the version info when the version is fully specified rather than pulling all versions from github. That probably requires a fair bit of reorganising the launching code so it's not a quick thing to do.
